tests/integration-e2e/test_cli_opts.py

```diff
@@ -165,3 +165,24 @@ def test_cli_output_path():
     assert not (test_folder_path / simconfig_output_path).is_dir(), \
            f"Directory '{simconfig_output_path}' should NOT exist."
     assert (test_folder_path / output_path).is_dir(), f"Directory '{output_path}' not found."
+
+def test_cli_report_buff_size():
```

Comment:
I wonder if this can be a unit test; doesn't look like it needs anything but a circuit; could just be the ringtest, no?

Reply:
This is using libsonatareport for its terminal output to check the buffer size is set, so I'll keep it here for now. In the future it would be better to have a simpler test and put that with the unit tests

Reply:
I think you do not need libsonatareport. You can just check SimConfig after neurodamus is called in python. This test is still useful to be sure that that value is really used in the end. 

TLDR: I argue for 2 tests, one unit test to check SimConfig value (probably you need to add 1 or 2 lines to an existing test) and one with libsonatareport to check that really, at the end, the value is used.

Reply:
ah right, good point - forgot that was the dependency.